### ISSUE TITLE
GPU Training broken on Conformer and Conformer with Gradient Accumulation

### ISSUE THREAD

**Issue by danielkope** [TensorSpeech/TensorFlowASR#47]
The transducer loss runs into a NaN and starts off already negative on the GPU (cuda10.2) - works fine on CPU.

Using supplied config - with small libri dataset.

**Comment by usimarit** [comment]
@danielkope Try reinstalling rnnt_loss for GPU, remember to `export CUDA_HOME=/usr/local/cuda` before running `scripts/install_rnnt_loss.sh`.

**Comment by danielkope** [comment]
That seems to have worked. So a library mismatch between TF <-> Cuda somehow made its way into my environment - interesting.
